hawk/src/main/java/com/orhanobut/hawk/Hawk.java

```diff
@@ -47,6 +51,10 @@ public static HawkBuilder init(Context context) {
       return remove(key);
     }
 
+    if (!isHawkInitialised()) {
+      return false;
```

Comment:
@carlonzo Everyone should use the support annotations ;-)
I understand what you mean.